syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.grizzhak.pixelpanels";
option java_outer_classname = "PanelRpcProto";
option objc_class_prefix = "HLW";

package pixelpanelrpc;

// The greeting service definition.
service PanelController {
  // Sends a greeting
  rpc PlayGif (PlayGifRequest) returns (PlayGifResponse) {}
}

// The request message containing the user's name.
message PlayGifRequest {
  string path = 1;
}

// The response message containing the greetings
message PlayGifResponse {
  string message = 1;
}